docs/baking.md

```diff
@@ -0,0 +1,182 @@
+<!--
+   - SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+# Baking with tezos-packaging on Ubuntu
+
+Tezos-packaging provides an easy way to install and set up infrastructure for
+interacting with Tezos blockchain.
+
+This article provides a step-by-step guide for setting up baking instance for Tezos on Ubuntu.
+
+## Prerequisites
+
+### Installing required packages
+
+In order to run a baking instance, you'll need the following Tezos binaries:
+`tezos-client`, `tezos-node`, `tezos-baker-<proto>`, `tezos-endorser-<proto>`.
+
+Currently used proto's are `007-PsDELPH1` (used on mainnet and delphinet) and
+`008-PtEdoTez` (used on edonet). Also, note that the corresponding ubuntu packages have protocol
+suffix in lowercase, e.g. list of available baker packages can be found
+[here](https://launchpad.net/~serokell/+archive/ubuntu/tezos/+packages?field.name_filter=tezos-baker&field.status_filter=published).
+
+To install them run the following commands:
+```
+# Add PPA with Tezos binaries
+sudo add-apt-repository ppa:serokell/tezos
+sudo apt-get update
+# Install packages
+sudo apt-get install tezos-client tezos-node tezos-baker-<proto> tezos-endorser-<proto>
+```
+
+Packages for `tezos-node`, `tezos-baker-<proto>` and `tezos-endorser-<proto>` provide
+systemd units for running the corresponding binaries in the background.
+
+### Downloading zcash-params
+
+Since v8.0 it's required to have the `sapling-output.params` and `sapling-spend.params` files from
+the `zcash-params` within the `XDG_DATA_DIRS` to run the `tezos-node`.
+
+We'll need these params to be accessible by different
+users (because `systemd` services are usually run as a separate user), thus the suggested way of getting
+them via [the zcash script](https://raw.githubusercontent.com/zcash/zcash/master/zcutil/fetch-params.sh), which will
+download them to the `HOME` directory of the current user, is not convenient. Moreover, it will
+download the large `sprout-groth16.params` which is unused at the runtime.
+So, it's better to download them to the `/usr/local/share/zcash-params`, the easiest way to do that
+is to run [`fetch-sapling-params.sh`](../scripts/fetch-sapling-params.sh) script:
+```
+sudo ./fetch-salping-params.sh
```

Comment:
```suggestion
sudo ./fetch-sapling-params.sh
```
Also, I wonder if we should provide an easier way to grab this, this kinda presumes that the repo has been cloned or at least the file downloaded, perhaps it would be better to include the command here that download an executes the script.

On the same point I guess is the link just above, but I think we'll cross that bridge when we'll know where this is going to end up.

Reply:
:facepalm: